neurogym/core.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, dt=100) -> None:
         self.t = self.t_ind = 0
         self.tmax = 10000  # maximum time steps
         self.performance = 0
-        self.rewards = {}
+        self.rewards: dict = {}
```

Comment:
why do some need a type hint, and others not? Could this be the (or a) reason for failing mypy?

Reply:
Usually type hints are inferred (that's why there are no hints in `self.tmax`, `self.performance`, etc.), but when initializing an empty dictionary mypy cannot infer what types the keys and values will have, so without the `dict` hint it gives an error.

Reply:
Then shouldn't the type hint include the types for the keys and values?

Reply:
it can but not necessary, with the dict only the mypy issue gets fixed